src/Symfony/Component/Messenger/Stamp/HandledStamp.php

```diff
@@ -41,32 +41,37 @@ public function __construct($result, string $callableName, string $handlerAlias
      * @param mixed $result The returned value of the message handler
      */
     public static function fromCallable(callable $handler, $result, string $handlerAlias = null): self
```

Comment:
as `$handlerAlias` could be null, if set, it must be:

```diff
- public static function fromCallable(callable $handler, $result, string $handlerAlias = null): self
+ public static function fromCallable(callable $handler, $result, ?string $handlerAlias = null): self
```

Reply:
FIxed :+1: